src/components/pages/consent-page.tsx

```diff
@@ -21,9 +22,7 @@ export function ConsentPage() {
       <p className="serlo-p">
         {replacePlaceholders(strings.consent.intro, {
           privacypolicy: (
-            <a className="serlo-link" href="/privacy">
-              {strings.entities.privacyPolicy}
-            </a>
+            <Link href="/privacy">{strings.entities.privacyPolicy}</Link>
```

Comment:
please revert. this is intentional since `/privacy` is and will not be handled by the frontend.

Reply:
We are currently having all this settings in one place within the `/content/link.tsx` file and `/privacy` is part of legacy already, so I thought it's fine to reuse this component here.

Reply:
since it's not a legacy link I'd like to make this more explicit. 
Otherwise I'm pretty sure I will remove the legacy logic one day without thinking about this :)